frontend/src/lib/api/candidate.ts

```diff
@@ -274,9 +283,9 @@ export const deleteAnswer = (answerId: string): Promise<Response | undefined> =>
 };
 
 /**
- * Get all the answers for the logged in user.
+ * Get all opinion answers for the logged in user.
  */
-export const getExistingAnswers = async (): Promise<Record<string, Answer> | undefined> => {
+export const getExistingOpinionAnswers = async (): Promise<Record<string, Answer> | undefined> => {
```

Comment:
We could do some renaming here: `getExistingOpinionAnswers` => `getOpinionAnswers` and apply the same logic elsewhere with 'existing' as well.